commands/logout.js

```diff
@@ -1,27 +1,27 @@
-'use strict';
-
 const cli = require('heroku-cli-util');
-const co = require('co');
 const child = require('child_process');
+const log = require('../lib/log');
 
 module.exports = function(topic) {
   return {
     topic: topic,
     command: 'logout',
     flags: [{ name: 'verbose', char: 'v', hasValue: false }],
-    description: 'Logs out from the Heroku Docker registry',
+    description: 'logs out from the Heroku Container Registry',
+    help: `Usage:
```

Comment:
the CLI already automatically adds a "usage" help bit. Just take the help attribute out here since it's not adding anything useful